File: src/main/java/BOJ2750.java
import java.io.IOException;
import java.util.ArrayList;
import java.util.List;
import java.util.Scanner;

public class BOJ2750 {

    /**
     * https://www.acmicpc.net/problem/2750
     * 수 정렬하기
     */

    public static void main(String[] args) throws IOException {
        Scanner scanner = new Scanner(System.in);
        int num = scanner.nextInt();
        List<Integer> numList = new ArrayList<>();

        for (int i = 0; i < num; i++) {
            System.out.println("input");
            int value = System.in.read();
            System.out.println("value = " + value);
            numList.add(value);
        }

        numList.stream()
                .sorted()
                .forEach(System.out::println);

    }

}

File: src/main/kotlin/BOJ2750.kt
import java.util.*

/**
 * https://www.acmicpc.net/problem/2750
 */

class BOJ2750 {

    fun run() = with(Scanner(System.`in`)) {

        val numOfInput = nextInt()
        val unsortedNums = IntArray(numOfInput)
        for (i in 0 until numOfInput) {
            val value = nextInt()
            unsortedNums[i] = value
        }

        for (i in unsortedNums.indices) {
            for (j in  0  until (unsortedNums.size - i - 1)) {
                if (unsortedNums[j + 1] < unsortedNums[j]) {
//                    println("${unsortedNums[j + 1]} < ${unsortedNums[j]}")
                    val temp = unsortedNums[j + 1]
                    unsortedNums[j + 1] = unsortedNums[j]
                    unsortedNums[j] = temp
                }
//                println("$i, ${unsortedNums.toList()}")
            }
        }

//        println("sorted Array")
        unsortedNums.iterator().forEach {
            println(it)
        }
    }

}


fun main() {
    BOJ2750().run()
}